nordic/trezor/scripts/build_sign_flash.sh

```
@@ -127,9 +129,15 @@ VERSION=$(get_version_from_file)
 
 # Update paths in signing and flashing commands
 if [ "$SIGN" -eq 1 ]; then
+    # zephyr.bin already contains a HEADER_SIZE-byte zero placeholder at offset 0
+    # (emitted by Zephyr linker via CONFIG_ROM_START_OFFSET). Strip it so that
+    # --pad-header does not prepend a second copy, which would push the vector
+    # table to slot_addr+2*HEADER_SIZE and cause an immediate hardfault on jump.
+    dd if=build/$APP_DIR/zephyr/zephyr.bin bs=1 skip=$((HEADER_SIZE)) \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)